src/types/apiResponseData.ts

```diff
@@ -0,0 +1,33 @@
+export interface PoolBalanceResponse {
+  ammPool: string
+  contractBalanceToken: number
```

Comment:
Be aware that numbers aren't accurate in javascript and this usually means they are not accurate in json either (depends on the implementation) -- rounding errors can and will happen. In general, it would probably be better to return strings.